lib/Wikia/tests/Service/User/UserPermissionsIntegrationTest.php

```diff
@@ -0,0 +1,293 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+use Wikia\Service\User\Permissions\PermissionsAccessor;
+
+class UserPermissionsIntegrationTest extends \WikiaBaseTest {
+	use PermissionsAccessor;
+
+	/**
+	 * @var int
+	 */
+	protected $staffUserId;
+
+	/**
+	 * @var int
+	 */
+	protected $testCityId;
+
+	/**
+	 * @var \User
+	 */
+	protected $staffUser;
+
+	/**
+	 * @var \User
+	 */
+	protected $anonUser;
+
+	/**
+	 * @var string
+	 */
+	const TEST_WIKI_NAME = "firefly";
+
+	/**
+	 * @var string
+	 */
+	const TEST_STAFF_USER_NAME = "PermissionTest";
+
+	protected function setUp() {
+		$this->staffUserId = \User::idFromName( self::TEST_STAFF_USER_NAME );
```

Comment:
We can - then this won't be an integration test but a unit test. I think both is useful in general.